### DISCUSSION TITLE
Confusion around how to join tables in supabase

### DISCUSSION THREAD

**Discussion by mohit2530** [supabase/supabase#11319] [Questions]
Hi everyone,

I am having some trouble around working with other tables and apologies this is very simple question but for some I can't get it right. To retrieve categories with category tags ( table below ) I wrote a query like such -

```
const {data, error} = await supabase.from('category')
.select(`
name,
id,
category_tag:container_location_id (id)
`);
```
**However, this results in an error.**

```
{
    "code": "PGRST200",
    "details": null,
    "hint": "Verify that 'category' and 'category_tag' exist in the schema 'public' and that there is a foreign key relationship between them. If a new relationship was created, try reloading the schema cache.",
    "message": "Could not find a relationship between 'category' and 'category_tag' in the schema cache"
}

```

```
create table category(
 id serial primary key not null,
name varchar(20) not null unique
);
```
```
create table category_tag(
id serial primary key not null,
tag_name varchar(10) not null unique,
container_location_id integer references category(id)
);
```

**Comment by GaryAustin1** [comment]
Do you have RLS on the category_tag table?

**Reply by mohit2530** [reply]
no i do not. I have RLS not enabled on both the tables.

**Reply by Jsmxing** [reply]
Why did I receive this message?

**Reply by mohit2530** [reply]
I have no idea. is like github broken ? I don't have you tagged though. apologies still.

**Comment by DarkAndromeda31** [comment]
For some reason I just got an email to supabase@noreply.github.com where I was CCed in with the message:
> ？什么鬼

Not sure how I got mentioned here, or at least this is where I was redirected to. Does anyone have an idea?

**Reply by Jsmxing** [reply]
？

**Reply by mohit2530** [reply]
not sure what this is, but apologies but didn't quite help resolve the problem. looks like a typo.
